themes/default/content/docs/intro/pulumi-service/billing-managers.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Billing Managers"
+meta_desc: Overview of Billing Managers in the Pulumi Service.
+menu:
+  intro:
+    parent: pulumi-service
+    weight: 11
+---
+
+The Billing Manager role provides customers the ability to have someone in your organization manage billing operations without granting them any additional permissions to view or modify your stacks, policies, or other organizational settings.
+
+Any organization admin can view, invite, and remove Billing Managers from their organization.
+
+Team Editions of the Pulumi Service are limited to 1 Billing Manager at a time. Enterprise and Business Critical subscriptions can have unlimited Billing Managers in their organization.
+
+## Inviting a Billing Manager to an Organization
+
+Billing Manager invitations work much the same way member invitations do, but they are managed on the Billing and Usage page.
+
+To invite a new Billing Manager:
+
+1. Select **Settings**.
+1. Select **Billing & usage**.
```

Comment:
```suggestion
2. Select **Billing & usage**.
```

Reply:
if we change this numbering, if we ever add future values we will have to adjust the numbering using all 1.s means this will be done automatically for us.